tests/datasets/base.py

```diff
@@ -20,3 +20,20 @@ def generate(num_rows):
             numpy.ndarray of size ``num_rows``
         """
         raise NotImplementedError()
+
+    @classmethod
+    def get_subclasses(cls):
```

Comment:
I notice we do this in a few places. Not really necessary for this PR, but maybe we should make this a utility function somewhere that can take any class and get all the subclasses

Reply:
Yeah that's what I had originally (put in a utils file), but I think @csala preferred it to be on the transformer. We can revisit this if it comes up again.